filelog.go

```diff
@@ -100,6 +108,8 @@ func (w *FileLogWriter) LogWrite(rec *LogRecord) {
 func (w *FileLogWriter) Close() {
 	close(w.rec)
 	<-w.completed
```

Comment:
In the future should we put everything into a waitgroup instead? I can understand not going down that rabbit hole now.